tests/agent_server/test_server_agent.py

```diff
@@ -303,8 +311,20 @@ async def test_server_restart(
     assert not resource_container.Provider.isset("agent1", "key3")
 
 
+@pytest.mark.parametrize(
+    "agent_deploy_interval",
+    ["2", "*/2 * * * * * *"],
```

Comment:
According to the documentation the format should be `min hour dom month dow`, but this expression contains 7 elements instead of 5?

Reply:
Updated the doc according to https://pypi.org/project/crontab/